### ISSUE TITLE
Key mappings with non-existent physical keys no longer work

### ISSUE THREAD

**Issue by nathanchance** [wez/wezterm#1519]
### What Operating System(s) are you seeing this problem on?

macOS

### WezTerm version

20220107-073311-4efcc81b

### Did you try the latest nightly build to see if the issue is better (or worse!) than your current version?

Yes, and I updated the version box above to show the version of the nightly that I tried

### Describe the bug

I use a [Planck EZ](https://www.zsa.io/planck) on my primary workstation, where I have mapped the "Lower" key + the arrow keys to Home, End, Page Up, and Page Down in the firmware to easily jump around in the terminal or my editor. To emulate that same key layout and behavior on my MacBook Air in `wezterm`, I have added `CMD` + the arrow keys mappings to my `wezterm` configuration (see below). With the latest nightly, these mappings no longer behave as I expect (the cursor no longer moves and the page does not scroll). I suspect this is related to #1483 but I did not want to hijack that thread. I tried to see how my mappings should be adjusted with those new changes but I could not figure it out. Perhaps this is because my MacBook Air keyboard does not actually have Home, End, Page Up, and Page Down keys (that's part of the reason I added the mappings)? I took a shot in the dark with the title, I can correct it if it turns out to be something else.

### Configuration

```
  keys = {
     {key="LeftArrow", mods="CMD", action={SendKey={key="Home"}}},
     {key="RightArrow", mods="CMD", action={SendKey={key="End"}}},
     {key="UpArrow", mods="CMD", action={SendKey={key="PageUp"}}},
     {key="DownArrow", mods="CMD", action={SendKey={key="PageDown"}}},
  },
```

### Logs

With `debug_key_events = true`:

```
 2022-01-07T16:57:50.779Z INFO  wezterm_gui::termwindow::keyevent     > key_event RawKeyEvent { key: Char('\u{f702}'), modifiers: SUPER, phys_code: Some(LeftArrow), raw_code: 123, repeat_count: 1, key_is_down: true, handled: Handled(false) }
 2022-01-07T16:57:50.779Z INFO  wezterm_gui::termwindow::keyevent     > Physical(LeftArrow) SUPER -> perform SendKey(KeyNoAction { key: Physical(Home), mods: NONE })
 2022-01-07T16:57:52.700Z INFO  wezterm_gui::termwindow::keyevent     > key_event RawKeyEvent { key: Char('\u{f703}'), modifiers: SUPER, phys_code: Some(RightArrow), raw_code: 124, repeat_count: 1, key_is_down: true, handled: Handled(false) }
 2022-01-07T16:57:52.700Z INFO  wezterm_gui::termwindow::keyevent     > Physical(RightArrow) SUPER -> perform SendKey(KeyNoAction { key: Physical(End), mods: NONE })
```

**Comment by wez** [comment]
Thanks for the report! This is fixed in 78c4c5642 which should be available to download within an hour.

FWIW, on a macbook, by default, Fn-LeftArrow is Home and Fn-RightArrow is End, Fn-UpArrow and Fn-DownArrow are PageUp/PageDown.

**Comment by nathanchance** [comment]
> Thanks for the report! This is fixed in [78c4c56](https://github.com/wez/wezterm/commit/78c4c5642595c7f93455ddb0f666e3145694b92e) which should be available to download within an hour.

Awesome, thank you so much for the quick fix!

> FWIW, on a macbook, by default, Fn-LeftArrow is Home and Fn-RightArrow is End, Fn-UpArrow and Fn-DownArrow are PageUp/PageDown.

Huh, I swear that I searched around for Home, End, Page Up, and Page Down key combinations before I wrote these mappings and I didn't find any... Thanks a lot for pointing those out (unfortunately, too used to the mappings I have now).
